README.md

```diff
@@ -271,6 +271,15 @@ Ripple checkout - A simple widget code to pay with XRP | [Link](https://github.c
 X-Address Information | [Link](https://xrpaddress.info/) | -
 
 
+---
+
+###  ♦ Merchants 
+
+&nbsp;&nbsp;&nbsp;&nbsp;Name&nbsp;&nbsp;&nbsp;&nbsp; | &nbsp;&nbsp;&nbsp;&nbsp;Link&nbsp;&nbsp;&nbsp;&nbsp; | &nbsp;&nbsp;&nbsp;&nbsp;Description&nbsp;&nbsp;&nbsp;&nbsp; 
+ --- | --- | --- 
+XRPL Merch | [Link](https://xrplmerch.com) | The go to shop for all your XRPL Clothing and Merch. Save 10% at checkout when you #PayWithXUMM.
+
```

Comment:
Change this in "description" in https://github.com/TusharPardhe/xrpl-cheat-sheet/blob/master/src/xrpl-cheat-sheet.json

Reply:
Have updated